skrub/_join_utils.py

```diff
@@ -40,6 +40,7 @@ def check_key(
         How to refer to `aux_key` in error messages.
     key_name : str
```

Comment:
Can you add the default also for `main_key_name` and `aux_key_name`

```suggestion
    key_name : str, default="key"
```